test_unstructured/cleaners/test_core.py

```diff
@@ -209,6 +209,30 @@ def test_group_broken_paragraphs_non_default_settings():
     )
 
 
+def test_group_broken_paragraphs_with_bullets():
+    text = """○The big red fox
```

Comment:
Do we really want to keep the ○ and · once we detected the element is inside a list?
If we want to clean the document probably deleting that characters could be better.

Reply:
Yes, because they're needed later to define the elements as list elements. I added a comment in the function noting that. I can add one to the test as well if needed

Reply:
My mistake! Looks like I removed the comment. I'll add it back noting as much